src/generate/renders/extraExt/CliExtMetadata.ts

```diff
@@ -4,20 +4,26 @@
  *-------------------------------------------------------------------------------------------- */
 import { EOL } from 'os';
 import * as path from 'path';
-import { isNullOrUndefined } from '../../../../utils/helper';
+import { isNullOrUndefined } from '../../../utils/helper';
 import {
     AzextMetadata,
     CodeGenConstants,
     PathConstants,
     ExtensionMode,
-} from '../../../../utils/models';
+    AzConfiguration,
+} from '../../../utils/models';
 import { CodeModelAz } from '../../CodeModelAz';
 import { TemplateBase } from '../TemplateBase';
+import * as nunjucks from 'nunjucks';
 
 export class CliTopMetadata extends TemplateBase {
-    constructor(model: CodeModelAz, isDebugMode: boolean) {
-        super(model, isDebugMode);
+    constructor(model: CodeModelAz) {
+        super(model);
         this.relativePath = path.join(model.AzextFolder, PathConstants.metadataFile);
+        this.tmplPath = path.join(
+            `${__dirname}`,
+            '../../../../src/templates/azext/azext_metadata.json.njx',
```

Comment:
can we put the base folder of the templates into a constant?

Reply:
sure

Reply:
changed it as a PathConstants.